version_mapping/3.1.0-4.0.0/example-instance-from-310.jsonld

```diff
@@ -60,7 +60,7 @@
 
         "ids:endpointHost" : {
 
-          "@type" : "ids:Host",
+          "@type" : "ids:Endpoint",
```

Comment:
```suggestion
          "@type" : "ids:Host",
```

It was actually Host in this previous version, so you must not change that here. Please revert, thank you.